consts/src/lib.rs

```diff
@@ -182,6 +198,23 @@ impl MessageBufferTrait for EdhocMessageBuffer {
             len: 0,
         }
     }
+
+    fn get(self, index: usize) -> Result<u8, AccessError> {
+        if index >= self.len {
+            return Err(AccessError::OutOfBounds);
```

Comment:
Why do you use `return`?

Reply:
Normally I prefer to halt and early-return upon errors, but I think in this case the function is small enough to use an else. Updated.